src/crypto/pem.h

```diff
@@ -13,8 +13,7 @@
 
 namespace crypto
 {
-  // Convenience class ensuring null termination of PEM-encoded certificates as
-  // required by mbedTLS
+  // Convenience class ensuring null termination of PEM-encoded certificates
```

Comment:
Do we still need to null terminate certs now? If not, we might as well get rid of this logic?

Reply:
I guess we probably do for compatibility with the current LTS.

Reply:
Probably not, but since this is a cleanup PR, I'd rather leave that to #3429. I'll add a comment on that issue.